src/components/forms/fields/CustomInput.tsx

```diff
@@ -38,8 +38,8 @@ const TextInput = (type: CustomInputType, inlineLabel: boolean) => ({
           id={id}
           type={type}
           required={required}
-          placeholder={placeholder ? formatMessage({ id: placeholder }) : ''}
-          label={inlineLabel ? formatMessage({ id: label }) : undefined}
+          placeholder={placeholder}
```

Comment:
`common/Input` component already to translation parth, `Fields/CustomInput` doesnt need to repeat this part